### DISCUSSION TITLE
How can I add script to code server pages?

### DISCUSSION THREAD

**Discussion by aqf671** [coder/code-server#5108] [Q&A]
In latest version, I found there are `error.html` and `login.html` but I wonder where is the `page.html`?

want to inject some custom script while page has been loaded.

btw, I tried to use nginx to inject script in head, but there is no effects, any ideas?

**Comment by jsjoeio** [comment]
Hmm...I'm not sure but maybe @code-asher has an idea?

**Reply by lg-nineoneninethreenine** [reply]
I don't really think there should be a page.html since TS renders the UI. Could be wrong, but I don't really know the use of a page.html when you have TS to control everything.

**Comment by code-asher** [comment]
There is no `page.html`.  The HTML for Code is found in `lib/vscode/out/vs/code/browser/workbench/workbench.html` I believe.
